### DISCUSSION TITLE
Some characters of Brownseya

### DISCUSSION THREAD

**Discussion by choess** [pteridogroup/ppg#13] [Q&A] [answered]
I started in on what I thought would be a simple housekeeping proposal (recognition of _Brownseya_ to maintain monophyly of _Pseudolycopodiella_) and as usual, nothing is simple to undertake.

[Chen et al. 2021](https://doi.org/10.1002/tax.12597) seems to be internally contradictory as regards the sporangia of _Brownseya serpentina_. "The non-monophyly of Lycopodiella sensu Øllgaard (1987) suggests that isovalvate sporangia shared by _Pseudolycopodiella caroliniana_ and _L. serpentina_ are homoplastic," but in the diagnosis of _Brownseya_, that genus is stated to have anisovalvate sporangia, which is given as one of the characters separating it from _Pseudolycopodiella_ s.s. [Burnard et al. 2016](https://www.researchgate.net/profile/Delaney-Burnard/publication/299361614_Phylogenetic_relationships_of_New_Zealand_Lycopodiaceae/links/5a9cb927aca2721e3f322484/Phylogenetic-relationships-of-New-Zealand-Lycopodiaceae.pdf) also report _L. serpentina_ as having isovalvate sporangia, which I presume is correct.

In that case, _Brownseya_ can be separated from _Pseudolycopodiella_ s.s. on the basis of phyllotaxy (10 ranks in the former, usually less in the latter, although _P. contexta_ and _P. krameriana_ get up to 8 or 9 ranks) and the visibility of sporophylls prior to dehiscence. I take it this latter character refers to the tendency of sporophylls in _Pseudolycopodiella_ s.s. to go from ascending to spreading as they mature. (Note that _P. limosa_, noted as similar to _B. serpentina_ but for which no molecular data is as yet available, belongs in _Pseudolycopodiella_ s.s. if these characters hold.)

Please let me know if I am correctly diagnosing the genera here and I would be happy to finish writing up the _Brownseya_ proposal.

**Comment by DrAshleyField** [comment, marked as answer]
The taxonomic diagnosis of _Brownseya_ from _Pseudolycopodiella_ in [Chen et al. 2021](https://onlinelibrary.wiley.com/doi/10.1002/tax.12597) is incorrect with regards to sporangial valves. The discussion of homoplasy of isovalvate sporangia between _Brownseya_ from _Pseudolycopodiella_, and then using anisovalvate sporangia as a diagnostic character of _Brownseya_ from _Pseudolycopodiella_ is internally contradictory. All material of _serpentina_ and _limosa_ that I have examined have isovalvate sporangia. The presence (or absence) of veinal mucilage canals (a feature of _Pseudolycopodiella_) was not examined.

The key characteristics for _Brownseya_ in Chen et al. 2021 are "Sporophylls arranged in alternating whorls of 5 or more, forming 10 or more longitudinal ranks; sporangia anisovalvate, axillary, hidden by free sporophylls until dehiscence=4" and "Tropophylls clearly longer than sporophylls"

In material of _serpentina_ I have examined, the sporangia are isovalvate (not anisovalvate), the sporangia and the sporophyll size range is nested in the trophophyll size range (not smaller than), the sporophyll arrangement ranges from alternating subwhorls of 3 to 5 [=longitudinal ranks of 6-10] (not as described) and the sporophylls begin to spread prior to sporangia dehiscence, making them visible.

Assuming _limosa_ were to be placed in _Brownseya_ (not yet combined), the sporangia are isovalvate (not anisovalvate), the sporophyll and trophophyll size would be more or less correct (not including the reduced trophophylls on the undersurface or peduncle), the sporophyll arrangement ranges from subworls of 3 to 5 [=6 to 10 longitudinal ranks] (not as described) and the sporophylls begin to spread prior to dehiscence of the sporangia (+the sporangia are visible prior to spread).

At present, I am unable to diagnose _Brownseya_ based on the characters presented by Chen et al. (2021). Also, I am unable to reproduce the tree topology of [Burnard et al. (2016)](https://link.springer.com/article/10.1007/s00606-016-1290-x) or Chen et al. (2021), but I am using a different sampling and marker set.

**Reply by choess** [reply]
Thank you, that is an extremely helpful and thorough answer. I will have to retract my intention of writing the proposal. I had hoped this would be a simple fix to achieve monophyly in the Lycopodielloideae, but my lack of familiarity with these species would not allow me to address this soundly. (I might play a bit with the tree topology, though.)

For those, like me, who were wondering if the veinal mucilage canals fell under the jurisdiction of the Engineers or the Post Office, Jim Bruce's 1976 paper [https://www.jstor.org/stable/2441916](https://www.jstor.org/stable/2441916) seems to be the definitive treatment of the subject. They lie abaxial to the vein when present; their presence seems to have been verified in the trophophylls of most or all of the North American diploid _Lycopodiella_ s.s. and their absence in _Pseudolycopodiella caroliniana_ and _P. meridionalis_.

**Reply by gonggashan88** [reply]
Hello, guys! Thanks for bringing attention to the aniso/isovalvate sporangia. It is true that the morphology of the few related species needs more attention in future. As we admitted in the paper, "the morphological features of the genus may need to be clarified in future." 

Well, distinguishing Brownseya from Pseuodolocypodiella did not seem to be a big issue, did it? The attention to the more subtle difference between Brownseya (L. serpentina) and (other species of) Lycopodiella had not been paid before our work. The paraphyly of Brownseya in relation to either Lycopodiella or Pseuodolocypodiella is quite convincing. If you were unable to reproduce our results about the paraphyly using your data, why not try to sample Burnard et al.'s (2016)] markers or our markers? It would be simple enough, wouldn't it?

In Chen et al. (2022) we said that Brownseya might contain 2-4 species and one of them might be L. limosa. Hope it is simple enough to find out by sequencing some loci of it.

Best to you all.

**Reply by choess** [reply]
For the record, I didn't mean to suggest that _Brownseya_ shouldn't be recognized; it's just that it is morphologically complex enough that I don't feel confident handling it on my own, although I thought in the beginning that I might be able to. I only cited _P. contexta_ and _P. krameriana_ because I started looking very carefully at the other morphological characters when the sporangial character broke down; comparing _Brownseya_ directly to sometimes obscure New World taxa is not something that I would expect to happen in real life, and I don't think that would have been reasonable to expect in the original paper.

I don't think it is fatal to _Brownseya_ if the diagnosis isn't quite clean at present; other than spore count, I don't think there are any gross morphological characters that unambiguously separate any _Myriopteris_ from any _Cheilanthes_ s.s. (as far as I know), and I don't think anyone is very troubled by that. And anyhow, it might get more clear once the presence or absence of mucilage canals is determined, for instance. I have seen _B. serpentina_ cited as a reason to lump rather than split in Lycopodielloideae, so I want to make sure whatever gets written into PPG II for circumscription stands up to refutation.

I just ran your Appendix S1 file though MegaX and _Brownseya_ came out where it should between _Lateristachys_ and _Palhinhaea_, so that's probably just a sampling issue.

**Comment by gonggashan88** [comment]
Thanks for your response and for running the data to confirm
the relationships of *Brownseya*. I personally can/prefer to
tolerate morphologically less clear diagnosis as long as evolutionary
evidence is strong. DNA data don't lie and are more objective, although
human errors are possible in data generation and analysis.

---
Best wishes, Libing
Li-Bing Zhang, Dr. nat., Curator, Missouri Botanical Garden
www.mobot.org/Zhang.shtml

**Reply by choess** [reply]
And I am sorry--I realized my email implicitly suggested Chen et al. 2021 was not good, which was not at all my intent, only to highlight my own limitations as someone not familiar with _Brownseya_ in person. I think it is not uncommon to have little discrepancies in the literature that specialists are aware of and the rest of us aren't, but this is a good opportunity to work on them! I would definitely prefer to be able to separate _Brownseya_ in PPG II rather than have a polyphyletic _Pseudolycopodiella_ or try to lump _Palhinhaea_ etc.

I am feeling more optimistic about developing a diagnosis--the other characters discussed from couplet 3 of Chen et al. are there to separate _Brownseya_ from _Lycopodiella_, so they're superfluous to the isovalvate/anisovalvate sporangium. Many American and African _Pseudolycopodiella_ have strongly anisophyllous leaves on the horizontal shoot, so it's really a few of those South American taxa that seem superficially closest to _Brownseya_. (I'm making heavy use here of Kelsey Cook's [unpublished thesis](https://etd.ohiolink.edu/apexprod/rws_etd/send_file/send?accession=miami1438305739&disposition=inline) on New World _Pseudolycopodiella_.) This may not be so bad to fix.

**Comment by gonggashan88** [comment]
No worries. Thanks for pointing out the isovalvate/anisovalvate issue we
missed. Hopefully more evidence will be uncovered in the near future.
Thanks for your service to the community by writing up proposals.

---
Best wishes, Libing
Li-Bing Zhang, Dr. nat., Curator, Missouri Botanical Garden
www.mobot.org/Zhang.shtml
